tests/plugins/test_store_metadata.py

```diff
@@ -60,7 +60,7 @@ def update_labels_on_build(build_id, labels):
      .with_args(**config_kwargs))
 
     workflow.user_params['namespace'] = 'namespace'
-    workflow.user_params['pipeline_run_name'] = 'store_metadata_test'
+    workflow.pipeline_run_name = 'store_metadata_test'
```

Comment:
workflow fixture already has pipeline_run_name set. Looks no need to set it again.

Reply:
updated